apps/docs/components/StepHikeCompact/index.tsx

```diff
@@ -29,7 +29,7 @@ const StepHikeCompact: FC<IStepHikeCompact> & IStepHikeCompactSubcomponents = ({
 
 const Step: FC<IStep> = ({ children, title, step }) => {
   return (
-    <div className="relative pb-8">
+    <div className="relative pb-8 last-step">
```

Comment:
Couldn't come up with anything better than `last-step`.

Reply:
we can use tailwind for this.
it would be something like:

`last:[&div][&div]-bg-transparent`

Reply:
great fix though, looks much better!!

Reply:
sorry this is it

`last:[&_div]:[&_div]:bg-transparent`

Reply:
`last:[&_div]:[&_div]:bg-transparent` doesn't work but if you rearrange the order to `[&_div]:[&_div]:last:bg-transparent` it does begin work. weird fix but finally working.